### PR TITLE
Update Chinese translation

### PR DESCRIPTION
**Pull request by Ate329**
Fixed some symbol and grammar mistakes and made the meaning of sentences more accurate.
Deleted the variables are not being used anymore.
I made a mistake in PR #541

### CODE REVIEW

File: scripts/installer-lang/zh.ps1 [hunk 1]
```diff
@@ -1,55 +1,49 @@
 [PSCustomObject]@{
     Welcome         = "
 ╔════════════════════════════════╗
-║ 欢迎使用 SpotX 的 Windows 版本 ║
+║ 欢迎使用 SpotX 的 Windows 版本  ║
```
Comment (amd64fox) [inline scripts/installer-lang/zh.ps1]: this space here is unnecessary

Comment (zyhe16) [reply scripts/installer-lang/zh.ps1]: ok i just changed it back

**Comment by amd64fox** [comment]
Thanks

**Comment by Ate329** [comment]
No problem!

**Comment by EdwardDK** [comment]
@amd64fox 


> Thanks

NerdEmojiNerdGIF

**Pull request merged**